client/src/components/Dropzone.tsx

```diff
@@ -56,12 +59,12 @@ const Dropzone = ({
       <div
         {...getRootProps({
           className:
-            'dropzone px-8 py-12 border border-dashed border-neutral-300 rounded-3xl text-center',
+            'dropzone p-8 border border-dashed border-neutral-300 rounded-3xl text-center h-56',
         })}
         tabIndex={-1}
         onClick={(e) => e.stopPropagation()}
       >
-        <div className='flex flex-col items-center gap-4'>
+        <div className='flex flex-col items-center justify-center gap-4 h-full'>
           <Upload size={40} className='text-neutral-500' aria-hidden />
```

Comment:
You might want to add "shrink-0" to `upload` icon so it doesn't disappear when resize and also the content goes outside the container border when resize.
![image](https://github.com/davidpak/SmartNote/assets/137979931/32ca6665-f106-4b26-b88c-7b39cee16218)